packages/salesforcedx-vscode-apex/snippets/apex.json

```diff
@@ -181,5 +181,91 @@
             "throw new ${1:Exception}($0)"
         ],
         "description": "Throw a new exception"
+    },
+    "List": {
+        "prefix": "List",
+        "body": [
+            "List<${1:object}> ${2:listName} = new List<$1>();"
+        ],
+        "description": "List constructor"
+    },
+    "map": {
+        "prefix": "Map",
+        "body": [
+            "Map<${1:object}, ${2:object}> ${3:mapName} = new Map<$1, $2>();"
+        ],
+        "description": "Map constructor"
+    },
+    "set": {
+        "prefix": "Set",
+        "body": [
+            "Set<${1:object}> ${2:setName} = new Set<$1>();"
+        ],
+        "description": "Set constructor"
+    },
+    "@AuraEnabled": {
+        "prefix": "@AuraEnabled",
+        "body": [
+            "@AuraEnabled",
+            "${1|public,global|} static ${2:string} ${3:methodName}(){",
+            "  $4",
+            "}"
+        ],
+        "description": "Aura Enabled Apex Method"
+    },
+    "AuraHandledException": {
+        "prefix": "auraHandledException",
+        "body": [
+            "throw new AuraHandledException('${1:exceptionText}');"
+        ],
+        "description": "Throw an exception for a lightning component to display"
+    },
+    "TestMethod": {
+        "prefix": "testMethod",
+        "body": [
+            "static testMethod void ${1:methodName}(){",
+            "  ",
+            "  test.startTest();",
+            "  system.RunAs(${2:contextuser}){",
+            "    ",
+            "  test.stopTest();",
+            "  ",
+            "}"
+        ],
+        "description": "Test method with runAs and stop/start"
+    },
+    "TestSetup": {
+        "prefix": "@TestSetup",
+        "body": [
```

Comment:
Similarly, we should put it so that the `@TestSetup` annotation is on its own line.